debug_toolbar/apps.py

```diff
@@ -206,3 +207,23 @@ def js_mimetype_check(app_configs, **kwargs):
             )
         ]
     return []
+
+
+@register()
+def debug_toolbar_namespace_check(app_configs, **kwargs):
```

Comment:
While the namespace issue is what the problem presents as, what we're really checking is if the toolbar is installed when running tests. This function name, doc string and changelog entry should reflect that.

Reply:
so instead of calling it namespace check I should call it debug_tolbar_installed_when_running_Tests